src/infra/envs/envs.go

```diff
@@ -2,9 +2,12 @@ package infraEnvs
 
 const (
 	InfiniteOsVersion            string = "0.1.2"
-	InfiniteOsBinary             string = "/infinite/os"
-	PersistentDatabaseFilePath   string = "/infinite/os.db"
-	TrailDatabaseFilePath        string = "/infinite/trail.db"
+	InfiniteOsMainDir            string = "/infinite"
+	InfiniteOsBinary             string = InfiniteOsMainDir + "/os"
+	PersistentDatabaseFilePath   string = InfiniteOsMainDir + "/os.db"
+	TrailDatabaseFilePath        string = InfiniteOsMainDir + "/trail.db"
+	MarketplaceItemsDir          string = InfiniteOsMainDir + "/marketplace"
+	ServicesItemsDir             string = InfiniteOsMainDir + "/services"
```

Comment:
Again, what kind of items? Is this where they are installed? Is this where the catalog is located?

Reply:
Changed to `MarketplaceCatalogItemsDir` and `ServiceInstalledItemsDir`.